rust/src/schema.rs

```diff
@@ -54,11 +60,20 @@ impl SchemaField {
     pub fn get_metadata(&self) -> &HashMap<String, String> {
         &self.metadata
     }
+
+    pub fn to_json(&self) -> Value {
```

Comment:
same here, we can leverage serde to avoid having to implement these `to_json` methods.